pyop2/global_kernel.py

```diff
@@ -363,7 +362,7 @@ def compile(self, comm):
                                 cppargs=cppargs,
                                 ldargs=ldargs,
                                 restype=ctypes.c_int,
-                                compiler=compiler,
+                                compiler=None,
```

Comment:
This is horrible. We only call `load` once so the API should be changed to avoid this.